x-pack/plugins/lens/common/expressions/metric_chart/metric_chart.ts

```diff
@@ -38,6 +38,24 @@ export const metricChart: ExpressionFunctionDefinition<
         defaultMessage: 'The chart title.',
       }),
     },
+    titleSize: {
+      types: ['string'],
+      help: i18n.translate('xpack.lens.metric.titleSize.help', {
+        defaultMessage: 'The chart title size.',
+      }),
+    },
+    titlePosition: {
+      types: ['string'],
+      help: i18n.translate('xpack.lens.metric.titlePosition.help', {
+        defaultMessage: 'The chart title position.',
+      }),
+    },
+    titleAlignPosition: {
+      types: ['string'],
+      help: i18n.translate('xpack.lens.metric.titleAlignPosition.help', {
+        defaultMessage: 'The chart title align position.',
+      }),
+    },
```

Comment:
@MichaelMarcialis I'm a bit confused because I don't see where in Lens you are using the words `title` and `subtitle` in the single metric visualizations. 
<img width="400" alt="Screenshot 2022-02-03 at 12 55 05" src="https://user-images.githubusercontent.com/1421091/152338897-be463754-b6df-45b1-b097-0bf129f2122a.png">
I see a `Display name` field that I can change to change the metric name, but I don't see any title or subtitle in any of our options. The only place I see the `title` word is in the CSS class name.
In the other metric vis (aggregated) I can see a `Custom label` that can be used to change the metric name.
<img width="400" alt="Screenshot 2022-02-03 at 12 56 22" src="https://user-images.githubusercontent.com/1421091/152339085-17e068a2-41b7-4478-9a14-5c52949f95e2.png">
This is actually badly aligned with the option to show and hide that name called `Show title`
<img width="400" alt="Screenshot 2022-02-03 at 12 56 17" src="https://user-images.githubusercontent.com/1421091/152339286-05ab37cb-b443-4093-adb8-a257f7bc4ac5.png">

The title is used on the `Axis` instead, where you can configure the `Axis title` in the top menu that changes the `Display name` of the vertical or horizontal axis.
<img width="381" alt="Screenshot 2022-02-03 at 13 12 39" src="https://user-images.githubusercontent.com/1421091/152340568-0ca1a1a5-570f-4709-b2c7-3bd7355b78a3.png">

When I think of a `title`, to me always refers to something more descriptive like: `Memory consumption in the last week` and is more suitable to be places on the top-left part of the visualization (aka panel title). A `label` instead, is used in various places and contains usually less information, just a tag: `RAM`, `CPU %`, really similar to what `name` is about.

As always is a matter of consistency across the interface and I don't see that neither with the term `title` or the `label`.
cc @gvnmagni